custom_components/powercalc/data/genio/I002334/model.json

```diff
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "TP-Link Tapo P110",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v0.22.1:docker"
+    },
+    "name": "genio_a60_9w_led_gls_bk",
```

Comment:
I prefer to have this without underscore and correctly cased. To make it consistent with all other lights. It will also be displayed in supported models list. https://github.com/bramstroker/homeassistant-powercalc/blob/master/docs/supported_models.md